test/e2e/scenarios/latency/scenario.go

```diff
@@ -0,0 +1,47 @@
+package latency
+
+import (
+	"time"
+
+	"github.com/microsoft/retina/test/e2e/framework/kubernetes"
+	"github.com/microsoft/retina/test/e2e/framework/types"
+)
+
+const sleepDelay = 5 * time.Second
+
+func ValidateLatencyMetric() *types.Scenario {
+	name := "Latency Metrics"
+	steps := []*types.StepWrapper{
+		{
+			Step: &types.Sleep{
+				Duration: sleepDelay,
+			},
+		},
```

Comment:
It might be nice if we eventually had the equivalent of `kubectl wait` with a context timeout. Not sure if it exists at the moment, but it's a task worth considering to fight future flakiness.

Reply:
the current framework we have is pretty extensible, I'm sure we could definitely add something 🙂